### ISSUE TITLE
dapr init complains about missing scheduler on darwin and linux arm64

### ISSUE THREAD

**Issue by mknj** [dapr/installer-bundle#26]
tar files: daprbundle_darwin_arm64.tar.gz  daprbundle_linux_arm64.tar.gz

```
./dapr init --slim --from-dir .
⌛  Making the jump to hyperspace...
⚠  Local bundle installation using --from-dir flag is currently a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.
ℹ️  Installing runtime version 1.14.0
❌  Extracting binaries and setting up components...
❌  error extracting scheduler binary: error open tar gz file dist/scheduler_linux_arm64.tar.gz: open dist/scheduler_linux_arm64.tar.gz: no such file or directory
```

side note: using the 1.13 cli the installation succeeds, as 1.13 does not know about the new scheduler

**Comment by sucharithv** [comment]
Same issue with the win 64 daprbundle_windows_amd64.zip V1.14

C:\dapr\daprbundle>dapr init --slim --from-dir .
Making the jump to hyperspace...
Local bundle installation using --from-dir flag is currently a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.
Installing runtime version 1.14.0
Extracting binaries and setting up components...
error extracting scheduler binary: error open zip file dist\scheduler_windows_amd64.zip: open dist\scheduler_windows_amd64.zip: The system cannot find the file specified.

**Comment by mikeee** [comment]
The installer bundle release is missing the scheduler, I'll get this followed up.
